Source/JavaScriptCore/runtime/JSGenericTypedArrayViewPrototypeFunctions.h

```diff
@@ -690,6 +729,60 @@ ALWAYS_INLINE EncodedJSValue genericTypedArrayViewProtoGetterFuncByteOffset(VM&,
     return JSValue::encode(jsNumber(thisObject->byteOffset()));
 }
 
+template<typename ViewClass>
+ALWAYS_INLINE EncodedJSValue genericTypedArrayViewProtoFuncForEach(VM& vm, JSGlobalObject* globalObject, CallFrame* callFrame)
+{
+    // https://tc39.es/ecma262/#sec-%typedarray%.prototype.foreach
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    ViewClass* thisObject = jsCast<ViewClass*>(callFrame->thisValue());
+    validateTypedArray(globalObject, thisObject);
+    RETURN_IF_EXCEPTION(scope, { });
+
+    size_t length = thisObject->length();
+
+    JSValue comparatorValue = callFrame->argument(0);
```

Comment:
I think the callback for `forEach` isn't comparator. So we should rename it `functorValue` ?

Reply:
Ah, fixed.